### ISSUE TITLE
Right click on label should offer "clear field"

### ISSUE THREAD

**Issue by koppor** [koppor/jabref#198]
I want to quickly clear a field while editing. I think, right click on a label and then "clear field" would be a good thing.

**Comment by MattLupino** [comment]
Hi, I'm interested in working on this enhancement. I have a small team of classmates to help me with trying to figure out how to do this for our class. If you have any advice (like relevant files or APIs to look at), we would appreciate it greatly.

Thanks!

**Comment by tobiasdiez** [comment]
Instead of the right click on label -> clear action that @koppor proposes, I would suggest to add a small "clear" button directly in the text box. The [controlsfx](http://fxexperience.com/controlsfx/features/#textfields) library already provides such a functionality and it should be easy to change the entry editor fields accordingly.

**Comment by MattLupino** [comment]
So, three weeks later, we have developed a change! Unfortunately, we didn't have the time/knowledge to find how to implement the clearable text fields, but we did create a new class and added an option in the right-click menus for all text fields with an option called "Clear field" that replaces all "text" in the text field with nothing (""). We will attempt to submit a pull request shortly. The pull request will come from one of our group members, [Weida](https://github.com/weidafan/).
